src/pages/quran-media-maker/index.tsx

```diff
@@ -99,6 +123,61 @@ const MediaMaker: NextPage<MediaMaker> = ({
     orientation,
   } = mediaSettings;
 
+  const dispatch = useDispatch();
+  const router = useRouter();
+
+  const queryParams = {
+    [QueryParam.SURAH]: String(surah),
+    [QueryParam.VERSE_FROM]: String(getVerseNumberFromKey(verseFrom)),
+    [QueryParam.VERSE_TO]: String(getVerseNumberFromKey(verseTo)),
+    [QueryParam.RECITER]: String(reciter),
+    [QueryParam.TRANSLATIONS]: String(translations),
+    [QueryParam.BACKGROUND_COLOR]: backgroundColor,
+    [QueryParam.OPACITY]: String(opacity),
+    [QueryParam.BORDER_COLOR]: borderColor,
+    [QueryParam.BORDER_SIZE]: String(borderSize),
+    [QueryParam.FONT_COLOR]: fontColor,
+    [QueryParam.VIDEO_ID]: String(videoId),
+    [QueryParam.QURAN_TEXT_FONT_SCALE]: String(quranTextFontScale),
+    [QueryParam.QURAN_TEXT_FONT_STYLE]: String(quranTextFontStyle),
+    [QueryParam.TRANSLATION_FONT_SCALE]: String(translationFontScale),
+    [QueryParam.ORIENTATION]: orientation,
+  };
```

Comment:
2 notes:

1. Do we need to use `String(...)` for most of the values, won't it be string by default?
2. If we need to, it seems `orientation`, `fontColor` and `borderColor` were missed.

Reply:
These values come from `mediaSettings`, so I have to convert number values to strings, for the `orientation`, `fontColor`, `borderColor`, and `backgroundColor` they are already strings.